test/components-utils/logger.spec.ts

```diff
@@ -0,0 +1,125 @@
+import {expect, sinon} from 'test-drive-react';
+import {overrideGlobalConfig, setGlobalConfig} from 'wix-react-tools';
+import {createLogger, error, errorOnce, warn, warnOnce} from '../../src/utils/logger';
+
+function assetError(fn: sinon.SinonSpy, err: Error) {
```

Comment:
Isn't this typo? Should be `assertError`, I think.

Reply:
thanks, fixed